db/migrate/20231208053131_create_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProjects < ActiveRecord::Migration[7.0]
+  def change
+    create_table :projects do |t|
+      t.string :name
+      t.text :description
+      t.text :image_url
+      t.date :starting_date
+      t.integer :duration
```

Comment:
I see the duration now - as mentioned above I would suggest end_date rather than duration, it's easier to work with